index.md

```diff
@@ -1,4 +1,6 @@
-
+<head>
+  <link rel="shortcut icon" type="image/png" href="https://d30y9cdsu7xlg0.cloudfront.net/png/83825-200.png"/>
```

Comment:
This won't be valid markup without a `title`.

Reply:
@omergulen thanks for the contribution. I'd like to add to @anthonydillon comment another request, the href should be a link to the actual file in the repository [here](images/icon/favicon.ico).